### ISSUE TITLE
Test Focus for Dropdown across browsers

### ISSUE THREAD

**Issue by LHinson** [patternfly/patternfly-react#981]
We have gotten feedback the following feedback regarding dropdown: 
> No focus indicator for the toggle button so there is no way for keyboard navigation users to know the item has focus

In this issue, we want to test the default browser focus for simple dropdown across all browsers. This issue includes identifying any issues for the react implementations and creating issue to address those issues.

**Comment by jgiardino** [comment]
Closing this one, since it's an issue with css in core.
